cmd/pkg/git.go

```diff
@@ -69,8 +69,11 @@ func Confirm(message string, args ...interface{}) {
 	Check(err)
 
 	answer = strings.TrimSpace(answer)
-	if answer != "y" {
+	if answer == "n" {
 		Fatalf("Aborting because your answer was: %s", answer)
+	} else if answer != "y" {
+		message = "Are you sure you want to proceed without creating a pre version first?"
```

Comment:
The message should be reused here! The dialogue is used by several pieces.

Also I would suggest to do this in a loop versus recursively :)